src/Http/Http.Abstractions/src/Metadata/IResult.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Http.Metadata
```

Comment:
Shouldnt be in the metadata namespace

Reply:
I was thinking the same thing. I thought I left a review comment. I moved it to `Microsoft.AspNetCore.Http`.